docs/packetcapture-guide.md

```diff
@@ -57,6 +57,9 @@ metadata:
 spec:
   fileServer:
     url: sftp://127.0.0.1:22/upload # Define your own sftp url here.
+    # Host public key (as a base64-endoded string) that will be accepted when connecting to the file server.
+    # If omitted, any host key will be accepted, which is insecure and not recommended.
```

Comment:
should we also document where/how the user could get this key normally? eg, 'usually you can find the key in the ~/.ssh/known_hosts file'. I feel it may not that well known.

Reply:
I added extra info. The known_hosts file would include that info if your machine has tried to connect to the server first. Otherwise this can be found in the SSH server configuration data.